go.mod

```diff
@@ -1,7 +1,8 @@
 module github.com/ava-labs/avalanche-cli
 
-go 1.21.12
-toolchain go1.22.5
+go 1.22
```

Comment:
Not sure if updating the golang version here was intentional or not. It is probably best left for it's own PR in case there is any unexpected behavior from updating.

We should also specify the full patch version as whatever the latest 1.22 release is when we do 👍